### DISCUSSION TITLE
Is it possible to export unified alerts to a format which can be used for provisioning?

### DISCUSSION THREAD

**Discussion by danfinn** [grafana/grafana#53914] [Alerting]
We are investigating moving to grafana 9 and unified alerts and one feature that seems to be missing that I think would make things significantly easier is the ability to export alerts created in the UI into a format that can be loaded into a yaml file for provisioning.  I'm spending some time now looking at the API to see if this is possible but having this easily done in the UI like it is for dashboards would be really nice.

Thanks,
Dan

**Comment by JohnnyQQQQ** [comment]
@danfinn thanks for reaching out. We got [this issue](https://github.com/grafana/grafana/issues/48622) where we track the effort for it. It's not scheduled yet, but we will eventually implement it.
